app/javascript/components/pages/DeckNew.js

```diff
@@ -8,32 +8,52 @@ export class DeckNew extends Component {
 			form: {
 				title: '',
 				description: '',
-				cards: [],
+				cards: [
```

Comment:
Does `form.cards` need to be initialized with empty cards (e.g. `{term: '', definition: ''}`)